neural_nets/activations/tests.py

```diff
@@ -1,43 +0,0 @@
-import numpy as np
```

Comment:
I deleted this file because it looks same as `plots.py`. We can refact our test process later.